SalesforceCommerceCloud/pwa-kit
# Change all ARIA labels to translatable messages

# Description

Noticed when reviewing #1570 that the ARIA labels were hard-coded strings, rather than translatable messages. Did a quick search in the project to find all the other instances of that. Spotted a few small issues along the way, so I fixed those as well.



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Listed in commit history

# How to Test-Drive This PR

To test that the messages are now translatable:
- Update relevant translations in translation files
- Start dev server
- Observe that your changes are reflected on the page

To test other changes
- Start dev server
- Click "My account" button as guest. Should open login modal.
- Click "My account" button as registered user. Should go to account.
- Hitting space/enter on "My account" button should have the same result as clicking.
- Hitting space/enter on account menu icon should open the menu.
- Turn on VoiceOver
- View login screen.
- Tab over the 👁️‍🗨️ icon. Should say "show password".
- Click 👁️‍🗨️ icon. Should show password and now say "hide password".
- Tab focus on "My account" button. It should now say "My account, button"
  - It should probably be a link, rather than a button, but that's harder to change without breaking customers
- Tab focus on account dropdown icon. It should now say "Open account menu"

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
